pkg/porter/explain.go

```diff
@@ -464,3 +465,38 @@ func (p *Porter) printDependenciesExplainTable(bun *PrintableBundle) error {
 		}
 	return printer.PrintTable(p.Out, bun.Dependencies, printDependencyRow, "Alias", "Reference")
 }
+
+func (p *Porter) printInstallationInstructionBlock(bun *PrintableBundle, bundleReference string) error {
+	fmt.Fprint(p.Out, "To install this bundle run the following command, passing --param KEY=VALUE for any parameters you want to customize:\n")
+
+	var bundleReferenceFlag string
+	if bundleReference != "" {
+		bundleReferenceFlag += " --reference " + bundleReference
+	}
+
+	// Generate any predefined credentials first
+	for _, credential := range bun.Credentials {
```

Comment:
We don't need to iterate for printing out how to generate credentials because when you run generate it will make a credential set that includes all of the credentials required by the bundle. So it can be a single statement that we only print out if the length of credentials is > 0.

The final command that we will tell them to run will be:

```
porter credentials generate mycreds -r REFERENCE
```

Normally if you leave off the name of the credential set, it will default to the name of the bundle. We could try to mimic that but I just checked and ... there's actually a potential bug in that code. So let's avoid that and just use a hard coded "mycreds" for the name in this command and in porter install. 😅